template/before.sh.erb

```diff
@@ -93,6 +93,12 @@ module purge
 export JUPYTERLAB_WORKSPACES_DIR=$PWD
 export jupyter_api="$JUPYTER_API"
 
+<%- if context.root_dir != "" -%>
+if [[ -r <%= context.root_dir %> ]]; then
+  NOTEBOOK_ROOT=<%= context.root_dir %>
+fi
+<%- end -%>
+
 # Notebook root directory
 export NOTEBOOK_ROOT="${NOTEBOOK_ROOT:-${HOME}}"
```

Comment:
🤦‍♂️ I think the user in the SNOW ticket just set `NOTEBOOK_ROOT` in their `~/.bashrc`. Let me check to see if that works.

Reply:
Yep. If I have something like `export NOTEBOOK_ROOT="/fs/ess/PZS0714/johrstrom"` in my `~/.bashrc` jupyter will respond to this environment variable.

Reply:
I can inform the user about this functionality, should I drop the PR? There's no software page for Jupyter afaict, I can make one and detail this functionality there. Then there is no need for the root directory selection to be in the job submission form, although it is slightly more cumbersome

Reply:
Yea I think this is a good enough workaround given this is the first request we've ever seen for this. And as you indicate, the form is already busy enough so avoiding 1 more thing I think is good.

A quick google search turns this up - 
https://www.osc.edu/vocabulary/documentation/jupyter

Reply:
😆 missed that, closing